lisa/sut_orchestrator/aws/platform_.py

```diff
@@ -158,11 +159,11 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
 
         self._aws_runbook = aws_runbook
         self._ec2_client = boto3.client("ec2")
```

Comment:
Remove this line? So every new instance will be created at needed.

`self._ec2_client = boto3.client("ec2")`